UnusedCatchParameterShouldBeUnnamed/Example1/README.md

```diff
@@ -0,0 +1,15 @@
+# Example1 Configs
+Make comment in PR:
```

Comment:
Please create separate PR to have update for new Checks from checkstyle, it will be merged very quickly